package.json

```diff
@@ -14,7 +14,7 @@
 		"ui"
 	],
 	"engines": {
-		"vscode": "^1.34.0"
+		"vscode": "^1.58.0"
```

Comment:
Upgrade minimum VSCode.  This one is still a year old so not that onerous of a requirement for our users